### ISSUE TITLE
- platform: homeassistant missing?

### ISSUE THREAD

**Issue by GrizzlyAK** [home-assistant/home-assistant.github.io#21524]
### Feedback

Should not the above platform be detailed on this page as options for trigger variables? There is a reference [here](https://www.home-assistant.io/docs/automation/trigger/#home-assistant-trigger), but I don't know where I learned that I could do the following:

"{{  trigger.platform == 'homeassistant' }}"

Just a thought.

### URL

https://www.home-assistant.io/docs/automation/templating/

### Version

2022.2.3

**Comment by frenck** [comment]
Hi @GrizzlyAK 👋 

I'm not sure what the suggestion is you are trying to make? The automation trigger variables are all documented, are they?

../Frenck

**Comment by GrizzlyAK** [comment]
Hi @frenck, on [this ](https://www.home-assistant.io/docs/automation/templating/) page is listed all of the trigger variable data available, and in each section there is a `trigger.platform` for `device`, `event`, `mqtt`, etc. I was just pointing out that nowhere on that page do I see where `trigger.platform` can be `homeassistant` as I use it in the example provided in the OP. Maybe this isn't the right place to include this information, but as mentioned, I only found a hint of it in the trigger docs. I think I learned about it on Discord. 

Maybe there are other values for `trigger.platform` not found here that I don't know about as well. I just thought it would be good to mention it here (or somewhere related). For example, isn't there at least a `start` and `shutdown` trigger event data object associated with `homeassistant`?

This all may seem obvious to the more HA-experienced user, or to developers, but in homage to my former self, it wasn't.
